packages/rulesets/src/spectral/test/path-for-tracked-resource-types.test.ts

```diff
@@ -0,0 +1,674 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+
+beforeAll(async () => {
+  linter = await linterForRule("PathForTrackedResourceTypes")
+  return linter
+})
+
+test("PathForTrackedResourceTypes should find errors for tracked resources having invalid path", () => {
+  const oasDoc = {
+    swagger: "2.0",
+    paths: {
+      "/subscriptions/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachine/{vmName}": {
```

Comment:
>/providers/Microsoft.Compute/virtualMachine/{vmName} [](http://example.com/codeflow?start=56&length=52)

Missing tests to cover the following

- /providers/Microsoft.Compute/virtualMachine/{vmName}
- /subscriptions/{subscriptionId}/providers/Microsoft.Compute/virtualMachine/{vmName}

Reply:
We have this test case covered in another rule's test suite.